readme.md

```diff
@@ -173,6 +174,8 @@ By default, a `<span class="icon icon-link"></span>` is used.
 When a function is given, it’s called with the current heading (`Element`) and
 should return one or more nodes.
 
+The hast can be hand-written from scratch, built using the `h` builder from [`hastscript`](https://www.npmjs.com/package/hastscript), or compiled from a string using the `fromHtml` builder from [`hast-util-from-html`](https://www.npmjs.com/package/hast-util-from-html).
+
```

Comment:
```suggestion
```

I recommend removing this here. Explaining all possible ways how to generate hast will take a lot of docs. The docs here are about how this option works, having too much there makes it harder to figure that out.